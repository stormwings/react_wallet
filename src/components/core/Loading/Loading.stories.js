import React from "react";

import { storiesOf } from "@storybook/react";

import { withKnobs } from "@storybook/addon-knobs";

import { jsxDecorator } from "storybook-addon-jsx";

import 'lilitcss/css/lilit.css';

import Loading from './Loading';


storiesOf("Dumb/Loading", module)
    .addDecorator(withKnobs)
    .addDecorator(jsxDecorator)
    .add("Loading", () => (
        <Loading />
    ));
