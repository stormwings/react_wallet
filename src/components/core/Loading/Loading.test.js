import React from 'react';

import { cleanup, render } from '@testing-library/react';

import renderer from 'react-test-renderer';

import Loading from './Loading';


afterEach(cleanup);


describe('<Loading />', () => {
    test('Matches the snapshot', () => {
        const tree = renderer
            .create(<Loading />)
            .toJSON();

        expect(tree).toMatchSnapshot();
    });

    test('Render correctly', () => {
        const { container: { firstChild } } = render(<Loading />);

        expect(firstChild).toBeTruthy();
    });
});
