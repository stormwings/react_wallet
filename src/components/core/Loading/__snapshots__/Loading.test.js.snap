// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> Matches the snapshot 1`] = `
<span>
  Loading...
</span>
`;
